.github/CONTRIBUTING.md

```diff
@@ -557,9 +557,11 @@ There is no strict process when it comes to merging pull requests. Pull requests
 
 * Make sure your pull request complies to the requirements outlined here
 
+* You are expected to have tested your pull requests if it is anything that would warrant testing. Text only changes, single number balance changes, and similar generally don't need testing, but anything else does. This means by extension web edits are disallowed for larger changes.
+
 * You are going to be expected to document all your changes in the pull request. Failing to do so will mean delaying it as we will have to question why you made the change. On the other hand, you can speed up the process by making the pull request readable and easy to understand, with diagrams or before/after data. Should you be optimizing a routine you must provide proof by way of profiling that your changes are faster.
 
-* We ask that you use the changelog system to document your change, which prevents our players from being caught unaware by changes - you can find more information about this [on this wiki page](http://tgstation13.org/wiki/Guide_to_Changelogs).
+* We ask that you use the changelog system to document your player facing changes, which prevents our players from being caught unaware by said changes - you can find more information about this [on this wiki page](http://tgstation13.org/wiki/Guide_to_Changelogs).
```

Comment:
Where do you draw the line here? Obviously github PRs are pointless in the changelog, but keeping a changelog is useful past the players knowing a change was made. I don't see any good reason we cannot log all changes, especially considering the bot uses the changelog tags to label PR's.

Reply:
If it's not a player facing change, then the 'changelogs' you're talking about should be descriptive commit messages. Anything else is a hacky workaround for the functionality already provided by git

Reply:
I mean we can pretend we live in a perfect world, but that doesnt really help us any.

Reply:
it's not for us you globalist

Reply:
The primary purpose of the changelog is to inform **players** and **game admins** about recent changes to the game.

In some cases, it makes sense to put a short line mentioning the existence of big refactor in the changelog if the probability that it broke something is high.

Coders and hosts are served by the Git commit log.

Reply:
Most of our coders are first and foremost players. I think it makes sense to put in the miniscule amount of work that updating the changelog in the case of a refactor or similar is to give the casual coder a summary of what has been merged lately. I doubt most of them even know how to browse the commit log.

Reply:
> Most of our coders are first and foremost players. 

But only a minority of our players are coders. We shouldn't spam non-coders with irrelevant technical information when what they want to know is whether a bug is fixed or a feature is added.

Reply:
Players are also submitting issue reports, and giving them a frame of reference to when shit changes can be useful.

But I see you are all against me on this, whatever.

Reply:
I will spend them by telling you to stop being such a baby and learn how to use git shortlog